client/allocdir/task_dir.go

```diff
@@ -142,15 +141,9 @@ func (t *TaskDir) Build(chrootCreated bool, chroot map[string]string, fsi cstruc
 // buildChroot takes a mapping of absolute directory or file paths on the host
 // to their intended, relative location within the task directory. This
 // attempts hardlink and then defaults to copying. If the path exists on the
-// host and can't be embedded an error is returned. If chrootCreated is true
-// skip expensive embedding operations and only ephemeral operations (eg
-// mounting /dev) are done.
-func (t *TaskDir) buildChroot(chrootCreated bool, entries map[string]string) error {
-	if !chrootCreated {
```

Comment:
Why get rid of this check?

Reply:
Hmm I guess because it's only called /w false.

Reply:
Indeed - `buildChroot` and `taskdir.Build()`[1] always is called with false, so didn't see the value.

[1] https://github.com/hashicorp/nomad/pull/5157/files/34ee0ba6b931f64dd5a97ca9df0c44506c17f07e#diff-301b205830d382e5f5ce9d97cd3e3404L78

Reply:
:+1: